scripts/Tegra/source_sync_6.0.sh

```diff
@@ -0,0 +1,344 @@
+#!/bin/bash
+
+# SPDX-FileCopyrightText: Copyright (c) 2012-2023 NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+# SPDX-License-Identifier: BSD-3-Clause
+#
+# Redistribution and use in source and binary forms, with or without
+# modification, are permitted provided that the following conditions are met:
+#
+# 1. Redistributions of source code must retain the above copyright notice, this
+# list of conditions and the following disclaimer.
+#
+# 2. Redistributions in binary form must reproduce the above copyright notice,
+# this list of conditions and the following disclaimer in the documentation
+# and/or other materials provided with the distribution.
+#
+# 3. Neither the name of the copyright holder nor the names of its
+# contributors may be used to endorse or promote products derived from
+# this software without specific prior written permission.
+#
+# THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS"
+# AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
+# IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE
+# DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT HOLDER OR CONTRIBUTORS BE LIABLE
+# FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL
+# DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR
+# SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER
+# CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY,
+# OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+# OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+
+
+#
+# This script sync's NVIDIA's version of
+# 1. the kernel source
+# from nv-tegra, NVIDIA's public git repository.
+# The script also provides opportunities to the sync to a specific tag
+# so that the binaries shipped with a release can be replicated.
+#
+# Usage:
+# By default it will download all the listed sources
+# ./source_sync.sh
+# Use the -t <TAG> option to provide the TAG to be used to sync all the sources.
+# Use the -k <TAG> option to download only the kernel and device tree repos and optionally sync to TAG
+# For detailed usage information run with -h option.
+#
+
+
+# verify that git is installed
+if  ! which git > /dev/null  ; then
+  echo "ERROR: git is not installed. If your linux distro is 10.04 or later,"
```

Comment:
We can remove the part about linux 10.04 right?
Too old

Reply:
Oh this file is taken from NVIDIA directly right?
If so ignore my last comment :)